<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="MPLWindow" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="mplVL">
       <item>
        <widget class="QWidget" name="verticalWidget" native="true">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <widget class="QPushButton" name="AddButton">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>140</y>
            <width>141</width>
            <height>32</height>
           </rect>
          </property>
          <property name="text">
           <string>Add line</string>
          </property>
         </widget>
         <widget class="QPushButton" name="QuitButton">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>140</y>
            <width>68</width>
            <height>32</height>
           </rect>
          </property>
          <property name="text">
           <string>Quit</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="Y_Angle_Start">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>30</y>
            <width>61</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="Z_Angle_Start">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>70</y>
            <width>61</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="HLidar">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>110</y>
            <width>141</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="Y_Angle_End">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>30</y>
            <width>61</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="Z_Angle_End">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>70</y>
            <width>61</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>10</y>
            <width>141</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Угол по Y</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>50</y>
            <width>141</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Угол по Z</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>90</y>
            <width>141</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Высота Lidar</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>10</y>
            <width>61</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Шаг по Y</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="Y_step">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>30</y>
            <width>41</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>50</y>
            <width>61</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Шаг по Z</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="Z_step">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>70</y>
            <width>41</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>763</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>QuitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>651</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
